src/controller/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const updatedUser = userService.updateUser({ id: +id, name });
+
+  if (!updatedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(updatedUser);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+  }
```

Comment:
In the 'create' function, if 'name' is not provided, the function will continue to execute after sending the 400 status. You should return after sending the status to prevent further execution.